_includes/html/header

```diff
@@ -4,6 +4,7 @@
 {% assign headerimage = include.headerimage %}
 {% endif %}
 <header id="top">
+    <a name="backtotop"></A>
```

Comment:
there is already a id="top" you can link to just above the <a> you added, we don't need a new one just link to the top of the header please.